src/annotator/test/integration/anchoring-test.js

```diff
@@ -53,12 +53,12 @@ describe('anchoring', function () {
 
   before(() => {
     guestImports.$mock({
-      './plugin/cross-frame': FakeCrossFrame,
+      './cross-frame': { CrossFrame: FakeCrossFrame },
     });
   });
 
   after(() => {
-    guestImports.$restore({ './plugins/cross-frame': true });
+    guestImports.$restore({ './cross-frame': true });
```

Comment:
Why not to restore all the imports like this `guestImports.$restore()`?

Reply:
Good catch. I'm guessing there was a reason for this originally but it no longer applies.